.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/task/methods/third_place/single_cell_perturbations"]
+	path = src/task/methods/third_place/single_cell_perturbations
+	url = https://github.com/okon2000/single_cell_perturbations/
```

Comment:
Please remove this submodule. Possible solutions are:

* If the resources you need are <10KB, copy the scripts you need to `src/task/methods/third_place/single_cell_perturbations/resources` and add them to `functionality.resources`
* Install the dependencies you need inside the Viash component itself:
  ```yaml
  platforms:
  - type: docker
    image: ...
    setup:
    - type: python
      packages: git+...
  ```
* Clone the repository inside your docker image
  ```yaml
  platforms:
  - type: docker
    image: ...
    setup:
    - type: apt
      packages: git
    - type: docker
      run: git clone https://github.com/okon2000/single_cell_perturbations/ /var/single_cell_perturbations
  ```

Reply:
done